File: app/src/main/java/com/example/seki/training_retrofit/MainActivity.java
package com.example.seki.training_retrofit;

import android.support.v4.app.FragmentTransaction;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        if (savedInstanceState == null) {
            final FragmentTransaction transaction = getSupportFragmentManager().beginTransaction();
            final WeatherFragment fragment = WeatherFragment.getInstance();
            transaction.replace(R.id.container, fragment);
            transaction.commit();
        }
    }
}

File: app/src/main/java/com/example/seki/training_retrofit/MainActivity.kt
package com.example.seki.training_retrofit;

import android.os.Bundle
import android.support.v7.app.AppCompatActivity

class MainActivity: AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        if (savedInstanceState == null) {
            val transaction = supportFragmentManager.beginTransaction()
            val fragment = WeatherFragment.getInstance()
            transaction.replace(R.id.container, fragment)
            transaction.commit()
        }
    }
}
